atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CQLKeyValueService.java

```diff
@@ -134,8 +132,8 @@ public Long apply(Entry<Cell, byte[]> entry) {
         }
     };
 
-    private Cluster cluster, longRunningQueryCluster;
-    private Session session, longRunningQuerySession;
```

Comment:
did you mean to do remove the split timeout clusters, or is github's diff just wonky?